spec/lib/tabular_form_builder_spec.rb

```diff
@@ -504,15 +504,21 @@
     subject(:output) { builder.label :name }
 
     it 'should output element' do
-      expect(output).to be_html_eql %{<label class="form--label" for="user_name">Name</label>}
+      expect(output).to be_html_eql %{
+        <label class="form--label"
+               for="user_name"
+               title="Name">
+               Name
+        </label>
+      }.squish
     end
 
     describe 'with existing attributes' do
-      subject(:output) { builder.label :name, 'Fear', class: 'sharknado' }
+      subject(:output) { builder.label :name, 'Fear', class: 'sharknado', title: 'Fear' }
 
       it 'should keep associated classes' do
         expect(output).to be_html_eql %{
-          <label class="sharknado form--label" for="user_name">Fear</label>
+          <label class="sharknado form--label" for="user_name" title="Fear">Fear</label>
```

Comment:
Well it does - right above in the output test.

Or do you mean the case where I pass in a blank value for the title?